readthedocs/settings/docker_compose.py

```diff
@@ -123,3 +123,5 @@ def DATABASES(self):  # noqa
     AZURE_EMULATED_MODE = True
     AZURE_CUSTOM_DOMAIN = 'storage:10000'
     AZURE_SSL = False
+
+    DATA_UPLOAD_MAX_NUMBER_FIELDS = 1000000
```

Comment:
This can also be None to drop the check

Reply:
I think locally we can completely remove it. I'll update the PR.